pymeasure/instruments/rohdeschwarz/fsl.py

```diff
@@ -257,3 +249,29 @@ def zoom(self, value):
                 passed it is interpreted as a frequency span.
             """
             self.write(f"FUNC:ZOOM {value}; *WAI")
+
+    # Channels -----------------------------------------------------------------
+
+    def create_channel(self, channel_type, channel_name):
+        strict_discrete_set(channel_type, ["PNOISE", "SANALYZER"])
+        self.write(f"INST:CRE:NEW {channel_type}, '{channel_name}'")
+
+    def show_channels(self):
```

Comment:
I actually think this should be a `Instrument.measurement` instead of a function with appropriate processing with `_channel_list_to_dict`.

Reply:
Resolved via [5bf1d62](https://github.com/bleykauf/pymeasure/pull/4/commits/5bf1d62757cc694692ff52d1cf04fdfb901b938b)